ws-nextjs-app/pages/[service]/new_live/[id]/Posts/styles.ts

```diff
@@ -0,0 +1,14 @@
+import { css, Theme } from '@emotion/react';
+import pixelsToRem from '../../../../../../src/app/utilities/pixelsToRem';
+
+export default {
+  postItem: ({ palette }: Theme) =>
+    css({
+      backgroundColor: palette.WHITE,
+      padding: `${pixelsToRem(16)}rem`,
```

Comment:
Is some UX work going on here or is this just a bit of placeholder styling?

Reply:
Yea all very much placeholder. Just wanted to add something to see any differences in theme provided CSS adding to the page weight.